docs/DATA_DICTIONARY.md

```diff
@@ -17,7 +17,7 @@ Field | Description | Exportable? | Reported by | Input Type
 **County** | Patient's county of residency | Exportable | Patient Reported | Text field
 **Referred by** | How did the patient found out about or was referred to the Fund | Exportable | Patient Reported | Dropdown
 **Special Circumstances** | What special circumstances is this patient dealing with at the time of the call? | Not Exportable | Patient Reported | Multiple checkboxes
-**LMP at intake** | How many weeks from a patient's Last Missed Period as reported during our first interaction. This is often a guess from the patient, but may also be an exact calculation from an ultrasound. We use LMP interchangably with 'weeks pregnant' in our funding contexts, even though this may be a different number in actuality. While DARIA automatically calculates LMP after this first interaction, LMP is editable if a patient receives a more accurate calculation later in the case management process. | Exportable | Patient Reported | Text field
+**Weeks along at intake** | How many weeks from a patient's Last Missed Period as reported during our first interaction. This is often a guess from the patient, but may also be an exact calculation from an ultrasound. We use LMP interchangably with 'weeks pregnant' in our funding contexts, even though this may be a different number in actuality. While DARIA automatically calculates LMP after this first interaction, LMP is editable if a patient receives a more accurate calculation later in the case management process. | Exportable | Patient Reported | Text field
```

Comment:
I suggest introducing the LMP acronym in parenthesis after the first time we write Last Missed Period. Like: "...Last Missed Period (LMP)".

For the rest of the wording, I second asking @lwaldsc for help!